nexus/db-queries/src/reconfigurator_chicken_switches.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Stray file?

Reply:
yup! good eye.

Reply:
Removed in [6add8b6](https://github.com/oxidecomputer/omicron/pull/8449/commits/6add8b654900ee77ac3f34f2a1f5a72d12fffd57)